mailscanner/conf.php.example.orig

```diff
@@ -0,0 +1,247 @@
+<?php
```

Comment:
This file was created when you merged the master branch, which was in conflict with some editing you've done, git advise you when this happen and you must fix this conflicts before committing the new code, welcome to git ;).